tools/audio.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python
+
+import os
+from time import sleep
+home = os.environ['HOME']
+
+while True:
+  while (os.system("ls /dev/video* 2>/dev/null")!=0):
```

Comment:
This will work, but only for the microphone on the usb camera. We want to support standalone microphones too, in the case of using a camera without a built in microphone.

So what we need to check is that a microphone is present, not a camera. Look for a command that lists alsa devices.